universe/ShipHull.h

```diff
@@ -137,26 +126,21 @@ class FO_COMMON_API ShipHull {
 
     //! Returns the condition that determines the locations where ShipDesign
     //! containing hull can be produced
-    auto Location() const -> const Condition::Condition*
-    { return m_location.get(); }
+    auto Location() const -> const Condition::Condition* { return m_location.get(); }
 
     //! Returns the names of other content that cannot be used in the same
     //! ship design as this part
-    auto Exclusions() const -> const std::set<std::string>&
-    { return m_exclusions; }
+    auto Exclusions() const -> const std::set<std::string>& { return m_exclusions; }
```

Comment:
Seem not related to the tags.  

This PR is already big, the minor tweaks make it harder to review. 

Please do not mix formatting and features in a single PR.  
Not sure if you do it manually or by tool, but in the first case, you could stash and do it in a different branch, and in the second just apply this to the whole project and commit in a separate branch before the next PR.

Reply:
manual